runners/direct-java/src/main/java/org/apache/beam/runners/direct/CopyOnAccessInMemoryStateInternals.java

```diff
@@ -355,6 +379,12 @@ private boolean containedInUnderlying(StateNamespace namespace, StateTag<? super
           public <KeyT, ValueT> MapState<KeyT, ValueT> bindMap(
               StateTag<? super K, MapState<KeyT, ValueT>> address,
               Coder<KeyT> mapKeyCoder, Coder<ValueT> mapValueCoder) {
+            if (mapKeyCoder == null || mapValueCoder == null) {
```

Comment:
Ah, I See. I'll move the exceptions from `CopyOnAccessInMemoryStateInternals` to `StateSpecs`.